Result-page/result.css

```diff
@@ -0,0 +1,10 @@
+.option{
+    color:var(--white-color)
+}
+
+.option-2{
+    background-color:var(--success-color)
+}
+.option-3{
+    background-color:var(--danger-color)
+}
```

Comment:
I think this is for correct and wrong answers , so that you can have an appropriate name for  `correct` and `wrong`.While adding **JS** , it will be easier to change the `bg` by adding these classes.